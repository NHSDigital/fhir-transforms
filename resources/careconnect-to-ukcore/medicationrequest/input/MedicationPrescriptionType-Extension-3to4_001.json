{"resourceType":"MedicationRequest","extension":[{"url":"https://fhir.nhs.uk/STU3/StructureDefinition/Extension-CareConnect-GPC-PrescriptionType-1","valueCodeableConcept":{"coding":[{"system":"https://fhir.nhs.uk/STU3/CodeSystem/CareConnect-PrescriptionType-1","code":"delayed-prescribing","display":"Delayed prescribing"}]}}]}
