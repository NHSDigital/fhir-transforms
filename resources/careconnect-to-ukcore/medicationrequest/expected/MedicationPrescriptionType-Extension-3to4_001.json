{
  "resourceType": "MedicationRequest",
  "courseOfTherapyType": {
    "coding": [
      {
        "system": "https://fhir.nhs.uk/STU3/CodeSystem/CareConnect-PrescriptionType-1",
        "code": "delayed-prescribing",
        "display": "Delayed prescribing"
      }
    ]
  }
}